src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -205,13 +206,15 @@ class C
 csharp_style_expression_bodied_properties = true:warning
 ";
 
-            VisualStudio.SolutionExplorer.AddFile(new ProjectUtils.Project(ProjectName), ".editorconfig", editorConfig, open: false);
+            // Adding an .editorconfig triggers file watchers and then asynchrony in the coding conventions library;
+            // wait until we see the appropriate change event triggered by it.
+            // Note: right now we watch for two changes, because the addition of the file triggers a design time build which
+            // causes https://github.com/dotnet/roslyn/issues/34309 to cause a spurious change.
```

Comment:
Fixing this isn't that hard, but keeping this to a test-only change means we can backport this to dev16.0. Once the bug is fixed I'll might remove the "numberOfChanges" flag since that's admittedly a hack.